lib/tasks/taxonomy/clear_descriptions.rake

```diff
@@ -1,9 +1,30 @@
 require 'taxon_description_updater'
+require 'description_remover'
+
 namespace :taxonomy do
   desc <<-DESC
     Clears all descriptions containing '...' or 'tbc'.
   DESC
   task clear_descriptions: [:environment] do
     TaxonDescriptionUpdater.new(%w[... tbc TBC ...tbc]).call
   end
+
+  desc <<-DESC
+    Clears description from taxon and taxons children recursively.
+  DESC
+  task clear_descriptions_for_branches: [:environment] do
+    DescriptionRemover.call(
+      %w[
+        /work
+        /housing-local-and-community
+        /health-and-social-care
+        /corporate-information
+        /going-and-being-abroad
+        /life-circumstances
+        /regional-and-local-government
+        /society-and-culture
+        /welfare
```

Comment:
Rather than hardcoding these branches, I think it would be better to make this task take a branch as an argument. Hopefully there will be descriptions written for these branches eventually, removing them is only a short term change.

Reply:
Sure